test/data/git/valid-commits.txt

```diff
@@ -1,3 +1,9 @@
+a2c164a718e45d7d3c5e37eec2008a3ebf371e36
+fix(testing): did some testing with breaking change
+
+BREAKING CHANGE: breaking change message generated from commitizen
```

Comment:
I think it's important to note that while this will mark commits as `breaking`, it won't be doing anything with the actual body of the commit (i.e. the part after `BREAKING CHANGE: `. We're keeping it in the `commit` object, but it's not being used in the writer. I haven't found a good, consistent way to expose that information, so I think that's beyond the scope of this PR, but I wanted to make sure that was understood.